swatch-metrics/deploy/clowdapp.yaml

```diff
@@ -280,6 +286,8 @@ objects:
                     key: psk
               - name: PROM_URL
                 value: ${PROM_URL}
+              - name: EVENT_RECORD_RETENTION
```

Comment:
Do we want to define `EVENT_RECORD_RETENTION` at the top of this file so it can be easily overridden in app-interface?

Reply:
Done.